### ISSUE TITLE
Wrong grid view height when embed in another view

### ISSUE THREAD

**Issue by rivera-ernesto** [spacenation/swiftui-grid#62]
Grid view gets compressed to about 20 when inside a ScrollView, and expands unnecessarily when inside a VStack.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/1261982/71379618-55a49300-25a2-11ea-9fd9-b28b4b5fbbbf.png">

<img width="383" alt="image" src="https://user-images.githubusercontent.com/1261982/71379634-63f2af00-25a2-11ea-9025-3cfacdfc6224.png">

Besides preview's layout is broken.

**Comment by ay42** [comment]
@rivera-ernesto this happens to all scroll based elements in SwiftUI.
```swift
ScrollView {
    List {
        Text("123")
        Text("123")
    }
}
```
Just like List, Grid has a scrollview inside and this becomes ScrollView inside a ScrollView.
I think the best approach here is to make Grid not-scrollable by default (Just like VStack). I'll start another issue to fix this.

**Comment by ay42** [comment]
@rivera-ernesto this is now fixed in version 1.0.0 with #80
